engine/src/flutter/testing/dart/fragment_shader_test.dart

```diff
@@ -311,6 +314,65 @@ void main() async {
     blueGreenImage.dispose();
   });
 
+  test('FragmentShader renders sampler with filter quality default', () async {
+    final FragmentProgram program = await FragmentProgram.fromAsset('texture.frag.iplr');
+    final Image blueGreenImage = await _createBlueGreenImage();
```

Comment:
I haven't seen the golden's yet, is the filtering effect visible enough with this?  I was thinking checker board would be easier to see.